src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => {
+  return (
+    <div className="movies">
+      {movies.map(movie => (
+        <MovieCard movie={movie} key={movie.toString()} />
```

Comment:
Why to you make string from movie?
You have `imdbId` in movie
```suggestion
        <MovieCard movie={movie} key={movie. imdbId} />
```